src/topics/inputs.md

```diff
@@ -143,13 +143,85 @@ The `baseCommand` field will always appear in the final command line before the
 
 [touch]: http://www.linfo.org/touch.html
 
+### Optional Input Parameters
+
+Optional `inputs` include `label` and `secondaryFiles`.
```

Comment:
Maybe start with Optional input parameters must include. As we have one `inputs` attribute, with optional entries (an input).